src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = sourceString
+    .split(';')
+    .filter((style) => style.includes(':'))
+    .reduce((prev, style) => {
+      const [key, value] = style.split(':');
```

Comment:
The 'split' function inside 'reduce' correctly splits each style into a key-value pair. Be cautious of potential issues if there are additional colons in the value part of the declaration.